lib/house.rb

```diff
@@ -0,0 +1,50 @@
+class House
+  attr_reader :address, :rooms
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+    @market_average = 500000
+  end
+
+  def price
+    (@price.delete("$")).to_i
```

Comment:
nice job catching that you had to write this method yourself instead of relying on an attr_reader

Reply:
you don't need parentheses here, as the part you have in parentheses is all one method call anyway